app/main.py

```diff
@@ -1,6 +1,35 @@
+import os
+from datetime import datetime as dt
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
```

Comment:
Code Style Issue: It's better to use constants for all the parts that are not changing in your URL. You should define a constant for the key parameter in your URL. Right now, you are only defining the base URL and the city as constants.